helix-view/src/editor.rs

```diff
@@ -529,15 +529,17 @@ impl Default for WhitespaceCharacters {
 }
 
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
-#[serde(default)]
+#[serde(default, rename_all = "kebab-case")]
 pub struct IndentGuidesConfig {
+    pub skip_first: bool,
     pub render: bool,
```

Comment:
Very nit-picky but I would like `pub render: bool` to be at the top of both the struct definition and in Default impl. Seems like the other structs typically starts with the enable/render flag so it's more consistent that way.

Reply:
makes sense! indeed more consistent